src-bindings/vscode/vscode.ml

```diff
@@ -2978,15 +2978,14 @@ module Window = struct
       [@@js.get "vscode.window.onDidCloseTerminal"]
 
     val showTextDocument :
-         document:
-           ([ `TextDocument of TextDocument.t | `Uri of Uri.t ][@js.union])
+         document:TextDocument.t
       -> ?column:ViewColumn.t
       -> ?preserveFocus:bool
       -> unit
       -> TextEditor.t Promise.t
       [@@js.global "vscode.window.showTextDocument"]
 
-    val showTextDocument2 :
+    val showTextDocument' :
```

Comment:
Could you describe the issue this commit is fixing? I think also putting it in a separate PR would be valuable.

Reply:
In VSCode extension API `showTextDocument` has a couple of overloads. Specifically the cases _without_ `TextDocumentShowOptions` only take `TextDocument` arguments, not `Uri`: https://code.visualstudio.com/api/references/vscode-api#window.showTextDocument.

I think there wasn't a problem before because the `Uri` overload is not used by OCaml Platform.